script.js

```diff
@@ -1,27 +1,47 @@
 const BurgerEl = document.getElementById("burger");
-const NavItemsEl = document.getElementById("navItems")
+const NavItemsEl = document.getElementById("navItems");
+const menuOpen = document.createElement("div");
+menuOpen.classList.add("menuOpen");
+const overlay = document.querySelector('#overlay');
+overlay.appendChild(menuOpen);
 
-function burgerToggle(){
-    if(NavItemsEl.style.visibility == "hidden"){
+function burgerToggle() {
+    if (NavItemsEl.style.visibility == "hidden") {
         NavItemsEl.style.visibility = "visible";
     }
-    else{
+    else {
         NavItemsEl.style.visibility = "hidden";
-    } 
+    }
 }
 
-BurgerEl.addEventListener("click", ()=>{    
-    NavItemsEl.classList.toggle("active")   
+BurgerEl.addEventListener("click", () => {
+    console.log('click open');
+    // NavItemsEl.classList.toggle("active")
+    overlay.style.display = "block";
+    menuOpen.innerHTML = `<span id='closeBtn'>&#10005;</span>
+    <ul id="menuNavItems">
+            <li class="menuNavItem">Shop</li>
+            <li class="menuNavItem">Recycle denim</li>
+            <li class="menuNavItem">Contact us</li>
+            <li class="menuNavItem">FAQ</li>
+        </ul>`
```

Comment:
Should there be a semicolon at the end of the line?

This creates a new DOM element every time the burger element is clicked? This could possibly be a performance issue in the future.

Is there a specific reason this is not rendered once into the document structure (if I understand the code correctly), and then the visibility toggled?